deployments/systemd/hooks.sh

```diff
@@ -122,3 +122,15 @@ function stop_k8s_pods() {
 	fi
 	return 0
 }
+
+# refresh-cdi triggers the nvidia-cdi-refresh service to regenerate CDI
+# specifications, making updated GPU devices available to container runtimes.
+function refresh-cdi() {
+    # Check if nvidia-cdi-refresh.service exists
+    if systemctl list-unit-files nvidia-cdi-refresh.service --quiet; then
+        echo "Found nvidia-cdi-refresh.service, calling systemctl..." >&2
+        if ! systemctl start nvidia-cdi-refresh.service; then
```

Comment:
Question: Should this be `restart` instead of `start`?

Reply:
You are right, will edit

Reply:
Done